cmd/bucket-notification-handlers.go

```diff
@@ -295,6 +296,9 @@ func (api objectAPIHandlers) ListenBucketNotificationHandler(w http.ResponseWrit
 		if !ok {
 			return false
 		}
+		if ev.S3.Bucket.Name != values.Get(peerRESTListenBucket) {
```

Comment:
Must use `values` instead of `r.URL.Query()` in `peer.Listen(..)` at https://github.com/minio/minio/pull/8677/files#diff-64acbec146a07c70d152644e0b92a3faR313 too